pages/index.html

```diff
@@ -133,9 +133,10 @@ <h3 class="version-profile--name">Travis Sanon</h3>
         <!-- End Apprentices 8.0 -->
 
         <div class="version-project">
-          <h4 class="version-project--heading">project</h4>
+          <h4 class="version-project--heading">projects contributed to</h4>
```

Comment:
The other apprentices just have a title of "project," so I'm wondering if it would be more consistent to use "projects" here

Reply:
It wouldn't be accurate, though.

Reply:
Approving the existing copy per our slack convo about Accessibility Audit site having many non-apprentices on it 👍